<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaRecuperarClave</class>
 <widget class="QWidget" name="PantallaRecuperarClave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="btnValidarCodigo">
     <property name="text">
      <string>Validar C칩digo</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Recuperar Contrase침a</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="labelResultado">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineCodigo">
     <property name="placeholderText">
      <string>C칩digo</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineNick">
     <property name="placeholderText">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEmail">
     <property name="placeholderText">
      <string>Correo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btnEnviarCodigo">
     <property name="text">
      <string>Enviar C칩digo</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEmail</tabstop>
  <tabstop>lineNick</tabstop>
  <tabstop>btnEnviarCodigo</tabstop>
  <tabstop>lineCodigo</tabstop>
  <tabstop>btnValidarCodigo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
